### PR TITLE
signalR no longer reloads on disconnect

### PR DESCRIPTION
**Pull request by sei-tspencer**

### CODE REVIEW

**Review by sei-npacheco — APPROVED** [review]

File: src/app/services/signalr.service.ts [hunk 1]
```diff
@@ -99,74 +99,74 @@ export class SignalRService {
     this.applicationArea = applicationArea;
     const accessToken = this.authService.getAuthorizationToken();
     this.hubConnection = new signalR.HubConnectionBuilder()
-      .withUrl(
-        `${this.settingsService.settings.ApiUrl}/hubs/main?bearer=${accessToken}`
-      )
-      .withAutomaticReconnect(this.retryArray)
+      .withUrl(this.getHubUrlWithAuth())
```
Comment (sei-aschlackman) [inline src/app/services/signalr.service.ts:102]: Just wanted to make sure you're aware of the [AccessTokenFactory](https://github.com/cmu-sei/Vm.Ui/blob/development/src/app/services/signalr/signalr.service.ts#L45) option. Not sure if it makes a practical difference. Does your getHubUrlWithAuth() get called on reconnects?

Comment (sei-tspencer) [reply src/app/services/signalr.service.ts:102]: I'm aware of it, but I didn't see any way to have it put the token in the URL, which is what we have to do.  It seems to only add it as an auth header.  Maybe I missed something?  Yes, getHubUrlWithAuth() gets called on the reconnects.

Comment (sei-tspencer) [reply src/app/services/signalr.service.ts:102]: Well, I should say, Reconnects call getHubUrlWithAuth() and updates the hubConnection.baseUrl

Comment (sei-aschlackman) [reply src/app/services/signalr.service.ts:102]: I believe putting the token in the URL is exactly what the accessTokenFactory does. Not a big deal, just might save a line or two of manually re-setting the baseUrl and should handle updating the token on automatic disconnect/reconnects too, I think.

**Pull request merged**